v0.9.0/quickstart.md

```diff
@@ -1,3 +1,7 @@
+---
```

Comment:
what's this for?  it's in the v0.9.0 dir?

Reply:
I think it's for #1126; switching versions while on a renamed page will result in a 404.